### ISSUE TITLE
`obsworker.service` does not start anymore on Tumbleweed hosts because `/etc/rc.status` has been dropped from `aaa_base` and `insserv-compat`.

### ISSUE THREAD

**Issue by ggardet** [openSUSE/open-build-service#11257]
# Issue Description
`obsworker.service` does not start anymore on Tumbleweed hosts because `/etc/rc.status` has been dropped from `aaa_base` in https://github.com/openSUSE/aaa_base/commit/a543e8e7db2a84ba56a6864d640ee68518cb116c in favor to `insserv-compat` which has been dropped from Tumbleweed.
This leads to the following error:
```
juin 18 09:01:08 hctw01 obsworker[6708]: /usr/sbin/obsworker: line 22: /etc/rc.status: No such file or directory
```

Expected Result
============
obsworker should be able to run on systemd only systems.

How to Reproduce
===============
Try to start obs worker on a Tumbleweed host.

**Comment by ggardet** [comment]
Ping.

**Comment by hennevogel** [comment]
@M0ses maybe you want to look into this?

**Comment by M0ses** [comment]
Fixed with bd489c7f16028f77c5783040b0f34569ea9a28c4
